person.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+require_relative 'nameable'
+
+# for person
+class Person
+  attr_reader :id
```

Comment:
- [ ] Please make sure the `Person` class inherits from `Nameable` class and has a method `correct_name` implemented, It should simply return the name attribute.
  - You can consider using the code:
  
```ruby
  class Person < Nameable
  .
  .

    def correct_name
       @name
    end
  .
  .
end